learn/guides/asyncapi-tool.md

```diff
@@ -16,144 +16,147 @@ The Ballerina AsyncAPI tool will make it easy for you to start the development o
 
 ```bash
-bal init
+bal asyncapi -i hello.yaml -o ./output_path
 ```
 
-3\. Create a new Ballerina file inside the folder (e.g., `slack_service.bal`) and paste the code below to it.
+Then, the generated files can be modified according to the custom requirements. When modifying the generated code segments, it will be easier to consider the below facts
+
+- All the incoming requests will be coming to the resource function in the dispatcher_service.bal file. Hence, if there is a necessity to add an authentication logic for the incoming calls, that logic can be included there before processing the incoming http request.
+- If more information is needed when initializing the listener like secrets, endpoint urls, tokens, refresh tokens and so on, update the init function in the listener.bal
+
+## Execute the Generated Sources
+
+In order to execute the generated Ballerina sources, please follow the below steps,
+
+1. Go inside the folder where the generated files exist
+2. Execute the command `bal init` - this will generate a Ballerina.toml file
+3. Create a new ballerina file inside the folder (Ex: slack_service.bal ) and use this piece of code
 
 ```ballerina
 listener Listener webhookListener = new(8090);
 
 service AppService on webhookListener {
-  remote function onAppMention(GenericEventWrapper event) returns error? {
-      //Implement the logic to use the received `event` here.
-  }
+   remote function onAppMention(GenericEventWrapper event) returns error? {
+       //implement the logic to use the received `event` here.
+   }
 }
 ```
 
-4\. Execute the command below from the root of the folder to execute this service.
-
-```bash
-bal run
-```
+That’s it, You can execute the command `bal run` to execute this.
 
 Below are some example libraries generated using the tool.
 
 | Module     | AsyncAPI specification                                                                                                                | Generated and modified code                                                                                              | Published module                                                                               |
 | ---------- | ------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------- |
 | **Slack**  | <a href="https://github.com/ballerina-platform/asyncapi-triggers/blob/main/asyncapi/slack/asyncapi.yml">`asyncapi.yml` of Slack</a>   | <a href="https://github.com/ballerina-platform/asyncapi-triggers/tree/main/asyncapi/slack">asyncapi-triggers/slack</a>   | <a href="https://central.ballerina.io/ballerinax/trigger.slack">ballerinax/trigger.slack</a>   |
```

Comment:
```suggestion
| **Slack**  | <a href="https://github.com/ballerina-platform/asyncapi-triggers/blob/main/asyncapi/slack/asyncapi.yml">`asyncapi.yml` of Slack</a>   | <a href="https://github.com/ballerina-platform/asyncapi-triggers/tree/main/asyncapi/slack">`asyncapi-triggers/slack`</a>   | <a href="https://central.ballerina.io/ballerinax/trigger.slack">`ballerinax/trigger.slack`</a>   |
```

Reply:
Thanks a lot for the above suggestions, resolved all.